src/components/Login/Login.js

```diff
@@ -50,69 +52,75 @@ export const Login = () => {
     }
   };
 
-  const validarUsuario = async (mail, password) => {
-    try {
-      const {
-        data: { data: { accessToken } = {} },
-      } = await validateLogin(mail, password);
-      setLocalStorageItem("accessToken", accessToken);
-      // toast.success("Usuario validado", {
-      //   position: toast.POSITION.TOP_RIGHT,
-      // });
-      navigate("/home");
-    } catch (e) {
-      toast.error("Contraseña o correo incorrectos", {
-        position: "top-center",
-        autoClose: 4000,
-        hideProgressBar: false,
-        closeOnClick: true,
-        pauseOnHover: false,
-        draggable: true,
-        progress: undefined,
-        theme: "light",
-      });
-    }
+  const validarUsuario = (mail, password) => {
+    setLoad(true);
+    setTimeout(async () => {
+      try {
+        const {
+          data: { data: { accessToken } = {} },
+        } = await validateLogin(mail, password);
+        setLocalStorageItem("accessToken", accessToken);
+        setLoad(false);
+        navigate("/home");
+      } catch (e) {
+        toast.error("Contraseña o correo incorrectos", {
+          position: "top-center",
+          autoClose: 4000,
+          hideProgressBar: false,
+          closeOnClick: true,
+          pauseOnHover: false,
+          draggable: true,
+          progress: undefined,
+          theme: "light",
+        });
+        setLoad(false);
+      }
+    }, 3000);
   };
 
-  return (
-    <Container>
-      <Wrapper>
-        <ToastContainer
-          closeButton={true}
-          position="bottom-right"
-          autoClose="3000"
-          hideProgressBar="true"
-        />
-        <Title>
-          <Link className="navbar-brand" to="/">
-            <img
-              src={require("../../assets/image-logo.png")}
-              width="100"
-              height="100"
-              alt="Logotipo Empresa"
-            />
-          </Link>
-        </Title>
-        <Form onSubmit={capturarDatos}>
-          <Field>
-            <FormFieldLabel for="mail">Correo electrónico</FormFieldLabel>
-            <Input type="text" name="mail" required />
-          </Field>
-          <Field>
-            <FormFieldLabel>Contraseña</FormFieldLabel>
-            <Input type="password" name="password" required />
-          </Field>
-          <Link1>
+  if (load === false) {
+    return (
+      <Container>
+        <Wrapper>
+          <ToastContainer
+            closeButton={true}
+            position="bottom-right"
+            autoClose="3000"
+            hideProgressBar="true"
+          />
+          <Title>
+            <Link className="navbar-brand" to="/">
+              <img
+                src={require("../../assets/image-logo.png")}
+                width="100"
+                height="100"
+                alt="Logotipo Empresa"
+              />
+            </Link>
+          </Title>
```

Comment:
i think this you could change it to this 
<img width="937" alt="image" src="https://user-images.githubusercontent.com/76448998/214905811-c7e91bc1-a4ce-4119-864c-89b0c5d21f99.png">

looks a little more clear 

```javascript
return load ? <Loader /> : <LoginForm />
```

and create a component only for the form and use props to comunicate to the parent component